Keyboard shortcuts (#237)
contao/docs: docs/manual/backend/backend-keyboard-shortcuts.en.md

---
title: "Keyboard shortcuts"
description: "To speed up the workflow when working with Contao, there are several keyboard shortcuts in the backend, 
with which certain commands can be called directly."

> url: "administration-area/keyboard-shortcuts"

Reviewer: Should we really also change the URL for the English version? Default would be the file path. I thought we would be using that.

Author: I would go with «administration-area/backend-keyboard-shortcuts». Why change the wording? [Past docs](https://docs.contao.org/books/manual/3.4/en/02-administration-area/index.html).

Reviewer: Ah I see. Hm, though we do write "back end" everywhere else (e.g. in the dev docs). Do you think we should change it to "back end" here too?

weight: 20
---

To speed up the workflow when working with Contao, there are several keyboard shortcuts in the backend that allow you to
call up certain commands directly. For example, the `Save` button is always located at the end of an input form, so that 

> it can only be clicked with the mouse when the page is completely was scrolled down.

Reviewer 2: ```suggestion
it can only be clicked with the mouse when the page is completely scrolled down.
```

> Normally this is not a problem, because while filling out a form you work from above anyway down through the side. But 

Reviewer 2: ```suggestion
Normally this is not a problem, because when filling out a form you work from top to bottom. But 
```

> if you later want to change only a single value at the beginning of the form, the Button not accessible without scrolling. 

Reviewer 2: ```suggestion
if you later want to change only a single value at the beginning of the form, the Button is not accessible without scrolling. 
```

> In this case, the experienced Contao user will be able to access the Keyboard shortcut `[Alt]+[S]` even without scrolling 

Reviewer: To keep it more simple and include even the "non-experienced Contao users" (😉) I would simply write.

```suggestion
In this case, you are able to access the Keyboard shortcut `[Alt]+[S]` even without scrolling 
```
